src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt

```diff
@@ -88,14 +88,14 @@ object Messages {
         "The one and only %name% has arrived, make way.",
         "%name%, did you bring the pizza, or was that someone else.",
         "Lord %name% has entered.",
-        "I don't know who %name% is but he sounds like a good guy.",
-        "You were the chosen one! It was said you would destroy TPH, not join them! You were my brother, %name%, I loved you!",
+        "I don't know who %name% is but they sound like a good person.",
+        "You were the chosen one! It was said you would destroy TPH, not join them! You were my sibling, %name%, I loved you!",
```

Comment:
"kin" might be a better word just for this, I don't think it implies such a strong family relation as "sibling" does, since they were not actually related.